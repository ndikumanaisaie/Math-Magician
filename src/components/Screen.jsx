/* eslint-disable react/require-default-props */
/* eslint-disable react/prefer-stateless-function */
import React from 'react';
import PropTypes from 'prop-types';

const Screen = ({ value }) => (
  <div className="screen">{value || '0'}</div>
);

Screen.propTypes = {
  value: PropTypes.string,
};

export default Screen;
