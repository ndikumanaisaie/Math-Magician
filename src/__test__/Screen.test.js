import React from 'react';
import renderer from 'react-test-renderer';

import Screen from '../components/Screen';

it('renders correctly the Screen component', () => {
  const screen = renderer.create(<Screen />).toJSON();
  expect(screen).toMatchSnapshot();
});
