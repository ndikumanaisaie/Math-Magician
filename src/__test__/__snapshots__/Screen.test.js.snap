// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the Screen component 1`] = `
<div
  className="screen"
>
  0
</div>
`;
